presto-native-execution/presto_cpp/main/types/tests/PlanConverterTest.cpp

```diff
@@ -99,13 +98,16 @@ class PlanConverterTest : public ::testing::Test {
  protected:
   static void SetUpTestCase() {
     memory::MemoryManager::testingSetInstance({});
+    registerPrestoToVeloxConnector(
```

Comment:
These tests are a bit odd. They register the connectors at different points in the test. I expected some error logic to be tested between the unregister and register or before the register, but there is none.

@majetideepak : Any comments ?

Reply:
Probably, a relic from the past. The tests cover the plan conversion and not the connectors.